mdoc/src/main/scala-3/mdoc/internal/markdown/Modifier.scala

```diff
@@ -0,0 +1,68 @@
+package mdoc.internal.markdown
+
+import mdoc.internal.markdown.Mod._
+
+/**
+  * A mdoc code fence modifier.
+  *
+  * Modifiers are parsed from code blocks like here
+  *
+  * ```scala mdoc:passthrough
+  * println("# Header")
+  * ```
+  *
+  * Currently, only supports parsing one modifier per code block.
+  */
+sealed abstract class Modifier(val mods: Set[Mod]) {
```

Comment:
Copied and some of the options remove - this is not used for worksheets, so possibly we could have removed everything almsot.